draft-ietf-ace-edhoc-oscore-profile.md

```diff
@@ -412,11 +412,6 @@ The EDHOC\_Information can either be encoded as a JSON object or as a CBOR map.
 |               |      |              | Suites   | suites             |
 |               |      |              | Registry |                    |
 +---------------+------+--------------+----------+--------------------+
-| key_update    | 3    | simple value |          | Support for the    |
-|               |      | "true" /     |          | EDHOC-KeyUpdate    |
-|               |      | simple value |          | function           |
-|               |      | "false"      |          |                    |
-+---------------+------+--------------+----------+--------------------+
 | message_4     | 4    | simple value |          | Support for EDHOC  |
```

Comment:
After this deletion, the values in the second column need to be renumbered. Also, it's an opportunity to fix the column headers by swapping "CBOR Type" and "CBOR value" :-)